s64da_benchmark_toolkit/prepare.py

```diff
@@ -105,21 +116,23 @@ def _run_shell_task(self, task, return_output=False):
 
             if return_output:
                 stdout, _ = p.communicate()
-                return stdout
+                print(f"output of run_shell_task: {stdout.decode('utf-8').strip()}")
```

Comment:
I think you should simply print out the output without any prefixes. `run_shell_task` doesn't mean anything to the end users, if anything they only expect to see the output from the command.
I would also remove the `strip()` both from here and from the return command below.

Reply:
done